protocol/protobuf/application_metadata_message.proto

```diff
@@ -78,9 +78,8 @@ message ApplicationMetadataMessage {
     SYNC_KEYCARD_ACTION = 63;
     SYNC_SOCIAL_LINKS = 64;
     SYNC_ENS_USERNAME_DETAIL = 65;
-    SYNC_FULL_KEYPAIR = 66;
```

Comment:
I just simply made changes in these 2 .proto files without taking care of backwardcompatibility just because we have not launched the MVP yet, once that's done being backward compatible will be mandatory. Hope you're ok with that.